Rakefile

```diff
@@ -37,6 +37,10 @@ if defined?(RSpec)
       t.pattern = "spec/processor_spec.rb"
     end
 
+    RSpec::Core::RakeTask.new(:detect_os) do |t|
+      t.pattern = "spec/helper/detect_os/*_spec.rb"
+    end
+
```

Comment:
@mizzy Yes, I'll do that after merge this and my other change because tests should have fixed before enable detect_os tests.